torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4547,6 +4547,16 @@ def _tensor(shape, dtype=dtype, low=None, high=None):
 
     return [SampleInput(tensor, args=args) for tensor, args in test_cases]
 
+def sample_inputs_one_hot(op_info, device, dtype, requires_grad, **kwargs):
+    def make_tensor_(shape, *, low, high):
```

Comment:
nit: things that end in `_` in PyTorch are by convention in-place operations. We should probably call this `_make_tensor` or something